### ISSUE TITLE
Documentation: JAX 101 value_and_grad tutorial example does not display returned gradients

### ISSUE THREAD

**Issue by FYC23** [jax-ml/jax#39140]
In the "Evaluating a function and its gradient using `jax.value_and_grad` section, the example assigns gradients to `Wb_grad` yet only prints out the loss value twice. It may be more helpful to change the code to print out `Wb_grad` too.

**Comment by kodlan** [comment]
hm indeed `Wb_grad` is not beeing printed, though it is aassigned. 
Created PR with a fix - #39405
Hopefully someone will approve it eventually :)
